.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
there must be a `ratio` type as well

Reply:
you're right, gonna add it now

Reply:
also, please add a README.md and say that it was taken from here https://github.com/solana-labs/solana-program-library/blob/5e320ba9768cca8ca751a4db26786ed13bb0d40f/token-lending/program/src/math/decimal.rs with some of our own additions

You can give some examples of how it's being used in Readme if you wish